stories/2023-02-01-Overwatch-League/OWL_story.ipynb

```diff
@@ -0,0 +1,3730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    df_elo
</pre><p> </p><p>Maybe pick a subset of columns and perhaps only one team in what's displayed here, so it's easier to see some Elo values increasing/decreasing with wins/losses?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>I would actually prefer to keep the whole table, because I think it helps understand how the data is structured (e.g. having one column per team). Does the plot below address your request?</p>